Added navbar and headline section

Bettenoch opened the pull request:
Added navbar with about portfolio and contact menu
Added hamburger and close menu icons
added logo 
Added headline section with description details and social media Icons

Bettenoch committed:
Added navbar and headline section

Added navbar with about portfolio and contact & headline section with description details

AdnanOlarmmi commented:
Hi @Bettenoch 😊,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Invalid Code Review Request 🆘💀
- [ ]  In the case of README consider not leaving any of the sections in the [Microverse README](https://github.com/microverseinc/readme-template) template as it's a project requirement.
Kindly complete your readme section, you can follow the readme in [this repo](https://github.com/AdnanOlarmmi/Technopreneur) for inspiration.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

Bettenoch committed:
Added about & work sections

Added about sections with projects
Added about sections with languages, skills and frameworks

Bettenoch committed:
Removed stylint errors

Bettenoch committed:
Updates Readme file

Daron976 commented on index.html:
```diff
@@ -0,0 +1,227 @@
+    <!-- Headline Section -->
+    <Section class="main_containers">
+        <div class="wrapper-grid">
+            <div class="gridcontainer">
+                <div class="snapshot">
+                    <img class="snapshot1" src="./assets/images/Snapshoot Portfolio.png" alt="snapshot">
+                </div>
+                <div class="grid-items">
+                    <h2>Tonic</h2>
+                    <ul class="snapshot-list content1">
+                        <li class="Company">CANOPY</li>
+                        <li class="description">Back End Dev</li>
+                        <li class="date">2015</li>
+                    </ul>
+                    <p class="snapshot-text">
+                        A daily selection of privately 
+                        personalized reads; no accounts
+                        or sign-ups required.
+                    </p>
+                    <ul class="snapshot-list content2">
+                        <li class="Company">HTML</li>
+                        <li class="description">CSS</li>
+                        <li class="date">Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
+                </div>
+                
+            </div>
```
[ ] Please note that you are required to resemble the Figma design as much as possible.
- The contents of your page are not properly aligned as in the Figma design
- There is too much space between your content and the edges
- The topic on your application is not as bold as it is on the Figma design
- The font of your description text is different from the Figma design
- The list of technologies on your application is not the same color as the Figma design
- The `see project` button on your application is different from the Figma design
- The text underneath the topic does not resemble the Figma illustrations

###
> This is the requirement
![Screenshot from 2023-06-03 17-02-23](https://github.com/Bettenoch/My_Portfolio/assets/69245974/e7b016d9-6b5e-4ee3-ba10-85fae7c939e7)

###
> This is your design
![Screenshot from 2023-06-03 16-51-39](https://github.com/Bettenoch/My_Portfolio/assets/69245974/473f4475-4676-445d-b0bb-6eeafc4b9d92)

###
> This is the Figma design
![Screenshot from 2023-06-03 16-46-29](https://github.com/Bettenoch/My_Portfolio/assets/69245974/c33163de-c027-4db7-92b3-2ba01689a287)

Daron976 commented on index.html:
```diff
@@ -0,0 +1,227 @@
+    <!-- About Section -->
+    <section class="main_containers">
+        <div class="about-container">
+            <div class="minicontainers about1">
+                <h1 class="about-primary-text">About Me</h1>
+                <p class="about-secondary-text">Hello I’m a software developer! I can help you 
+                    build a product , feature or website. 
+                    Look through some of my work and experience! 
+                    If you like what you see and have a project 
+                    you need coded, don’t hesitate to contact me.
+                </p>
+                <div class="social_media">
+                    <h2 class="tertiary-text">Let's connect</h2>
+                    <ul class="socialmedia-icons">
+                        <li><a href="#"><img src="./assets/images/github.svg" alt="github"></a> </li>
+                        <li><a href="#"><img src="./assets/images/Linkedin icon.svg" alt="Linkedin"></a> </li>
+                        <li><a href="#"><img src="./assets/images/angel list.svg" alt="angellist"></a> </li>
+                        <li><a href="#"><img src="./assets/images/medium.svg" alt="Medium"></a> </li>
+                    </ul>
+                  </div>
+                  <button class="btn">Get my resume</button>
+            </div>
```
[ ] Please note that as explained before you are required to follow the Figma design as much as possible.
- In the About section your design has a border around it which is not present in the Figma design
- The social media icons are a different color or shade to the Figma design
- The `get my resume` button is different from the one in the Figma design

###
> This is your design 
![Screenshot from 2023-06-03 17-05-23](https://github.com/Bettenoch/My_Portfolio/assets/69245974/d73c891a-2c9c-4ca8-9341-c5522f3727f2)

###
> This is the Figma design
![Screenshot from 2023-06-03 17-04-03](https://github.com/Bettenoch/My_Portfolio/assets/69245974/831f2ddd-44db-4d05-ba49-4c5a3e61ef87)

Daron976 commented on index.html:
```diff
@@ -0,0 +1,227 @@
+                  </div>
+                  <button class="btn">Get my resume</button>
+            </div>
+            <div class="minicontainers about2">
+                <ul class="skills">
+                    <li class="skills-title">
+                        <h3 class="skills-primary-text">Languages</h3>
+                        <img class="dropdown" src="./assets/images/dropdown-bottom.svg" alt="">
+                    </li>
+                    <li>
+                        <ul class="language-components">
+                            <li>
+                                <span><img class="logos javascriptlogo" src="./assets/images/javascriptimage.svg" alt=""></span>
+                                <p>Javascript</p>
+                            </li>
+                            <li>
+                                <span><img class="logos htmllogo" src="./assets/images/Htmlimg.svg" alt=""></span>
+                                <p>Html</p>
+                            </li>
+                            <li>
+                                <span><img class="logos csslogo" src="./assets/images/cssimg.svg" alt=""></span>
+                                <p>Css</p>
+                            </li>
+                        </ul>
+                    </li>
+                    <li class="skills-title-border">
+                        <h3 class="skills-primary-text">Frameworks</h3>
+                        <span><img src="./assets/images/dropdownleft.svg" alt=""></span>
+                    </li>
+                    <li>
+                        <h3 class="skills-primary-text">Skills</h3>
+                        <span><img src="./assets/images/dropdownleft.svg" alt=""></span>
+                    </li>
+                </ul>
+            </div>
```
[ ] Please note that your design is required to match the Figma design.
- The `languages` does not resemble the Figma design 
- The `frameworks and skills do not resemble the Figma design 
- There is a border in your application that is not present in the Figma design (when other students click on a section on Figma, there is a border created, please be careful and make sure any border is added is in the Figma design)

###
>This is your design
![Screenshot from 2023-06-03 17-11-44](https://github.com/Bettenoch/My_Portfolio/assets/69245974/2251a828-30fa-49a8-b6db-8641de0821eb)

###
> This is the Figma design
![Screenshot from 2023-06-03 17-12-20](https://github.com/Bettenoch/My_Portfolio/assets/69245974/e75a933d-b571-4615-8b2e-57baf73e9e42)

Daron976 commented on index.html:
```diff
@@ -0,0 +1,227 @@
+                
+            </div>
+
+            <div class="gridcontainer">
```
[ ] Please try to minimize the use of `div` tags and instead use more semantic tags such as `article`. In this instance, an `article` is a better option than a `div` because it tells the browser that this section is an important section and gives that section more meaning as opposed to a `div` tag - this will improve semantics which help with SEO.

Bantealem reviewed (commented):
Hi @Bettenoch 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that README is not professional. Please use the correct README template. Kindly remove the live demo section and set the proper `MIT` License.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Daron976 commented on index.html:
```diff
@@ -0,0 +1,227 @@
+                        <li class="description">CSS</li>
+                        <li class="date">Javascript</li>
+                    </ul>
+                    <button class="btn">See Project</button>
```
[ ] Please note that these buttons must follow the button styling outlined by the Figma design.

###
> This is the Figma
![Screenshot from 2023-06-03 17-28-20](https://github.com/Bettenoch/My_Portfolio/assets/69245974/0f56de3e-b97f-40f7-b36a-1bc36286c79c)
 design

Bettenoch committed:
Edited Readme.file

Bettenoch committed:
Update README.md

shahadat3669 reviewed (changes requested):
Hi @Bettenoch , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

<img src="https://media1.giphy.com/media/jmqAUNFXAElgafPK0g/giphy.gif"/>

## Invalid Code Review Request

You have submitted a project, that has an invalid readme file. Please take your time and update it according to the [template](https://github.com/microverseinc/readme-template).

If you use "VS Code" you can follow this [article ](https://code.visualstudio.com/docs/languages/markdown#:~:text=You%20can%20view%20the%20preview,real%2Dtime%20as%20you%20edit.)to update your readme file.

| Your | Professional |
| --- | --- |
|  ![image](https://github.com/Bettenoch/My_Portfolio/assets/55840999/7b9196d6-235b-4d34-a96a-64fbd532f3cb) | ![image](https://github.com/Bettenoch/My_Portfolio/assets/55840999/18309a2d-2a82-4f13-8908-3988150778f1) |

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Daron976 reviewed (changes requested):
Hi @Bettenoch , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:heavy_check_mark: Linters are passing
:heavy_check_mark: README file is professional
:heavy_check_mark: Correct Github flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- [ ] Please note that your README file has `...` below `Clone this repository to your desired folder or download the Zip folder:` which makes the preview of your README show all the raw data that should be masked 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

shella12 reviewed (commented):
Hi @Bettenoch, 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that does not have a professional README. As suggested by the previous reviewers please use the [Microverse template](https://github.com/microverseinc/readme-template) for your README. and use mark ups to make it look professional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

Bettenoch committed:
updated readme file

Bettenoch committed:
Update README.md

Bantealem reviewed (commented):
Hi @Bettenoch 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a Github conflict. Please resolve the conflict and request again. You can find the tutorial here on [how to resolve GitHub conflict here](https://youtu.be/xNVM5UxlFSA)

![image](https://github.com/Bettenoch/My_Portfolio/assets/101523582/83340b3b-4e87-4250-a4b0-03ce64d54956)


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

bienvenuushindi reviewed (commented):
Hi @Bettenoch 

Kindly make sure you fix the conflict as requested by the previous reviewer :arrow_down: 

> Hi @Bettenoch
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> ## Invalid Code Review Request
> 
> You have submitted a project, that has a Github conflict. Please resolve the conflict and request again. You can find the tutorial here on [how to resolve GitHub conflict here](https://youtu.be/xNVM5UxlFSA)
> 
> ![image](https://user-images.githubusercontent.com/101523582/243129931-83340b3b-4e87-4250-a4b0-03ce64d54956.png)
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._
